src/components/RocketList.js

```diff
@@ -1,9 +1,30 @@
 import React from 'react';
+import { Button } from '@material-tailwind/react';
+import PropTypes from 'prop-types';
 
-const RocketList = () => (
-  <>
-    <h1>From Rocket List</h1>
-  </>
+const RocketList = ({
+  rocketImage, rocketName, rocketDescription,
+}) => (
+  <div className="flex gap-6">
+    <img
+      className="w-1/5 sm:w-1/2 h-fit"
+      src={rocketImage}
+      alt={rocketName}
+    />
+    <div className="text-left">
+      <h3 className="font-bold">{ rocketName }</h3>
+      <p className="my-2">{ rocketDescription }</p>
+      <Button>
+        Reserve Rockets
+      </Button>
+    </div>
+  </div>
 );
 
+RocketList.propTypes = {
+  rocketImage: PropTypes.string.isRequired,
+  rocketName: PropTypes.string.isRequired,
+  rocketDescription: PropTypes.string.isRequired,
+};
```

Comment:
So according to the above, this will be in the `Rocket.js`, and here you can have something like this:

```
RocketList.propTypes = {
  rockets: PropTypes.arrayOf(PropTypes.shape({
    rocketId: PropTypes.string,
    rocketName: PropTypes.string,
    rocketDescription: PropTypes.string,
  })).isRequired,
};
```

We may need the `rocketId` for event handling, so keep it until we figure it out

Reply:
Ok! I never knew prop types could be validated in this form! This is added knowledge for me! Thanks!